x/distribution/alias.go

```diff
@@ -39,8 +38,7 @@ var (
 
 	NewMsgSetWithdrawAddress          = types.NewMsgSetWithdrawAddress
 	NewMsgWithdrawDelegatorReward     = types.NewMsgWithdrawDelegatorReward
-	NewMsgWithdrawDelegatorRewardsAll = types.NewMsgWithdrawDelegatorRewardsAll
-	NewMsgWithdrawValidatorRewardsAll = types.NewMsgWithdrawValidatorRewardsAll
+	NewMsgWithdrawValidatorCommission = types.NewMsgWithdrawValidatorCommission
```

Comment:
Is there any cost to just making the comission deposit directly into their own account? Its a separate merkle tree, so that may hurt caching a bit, but may be worth it for removing a message type

Reply:
Accounts don't have decimal coins balances - I think we probably want the extra precision for validator commission.

Reply:
Will it need more precision than 10^-9?